backend/src/main/java/com/shipping/backend/entities/PackageTypeResponse.java

```diff
@@ -0,0 +1,40 @@
+package com.shipping.backend.entities;
+
```

Comment:
tip: Take a look to lombok library to avoid boilerplate code.
[https://projectlombok.org](url)

Reply:
I'm adding lombok on the next history it was something that I already comment with Marin last saturday